package.json

```diff
@@ -19,17 +19,18 @@
   },
   "devDependencies": {
     "aegir": "^19.0.5",
+    "async-iterator-all": "^1.0.0",
     "chai": "^4.2.0",
-    "cids": "~0.5.5",
+    "cids": "^0.7.1",
     "go-ipfs-dep": "~0.4.17",
-    "ipfsd-ctl": "~0.39.2"
+    "ipfsd-ctl": "~0.44.0"
   },
   "dependencies": {
-    "async": "^2.6.1",
-    "ipfs-api": "^24.0.2",
-    "multiaddr": "^5.0.0",
-    "peer-id": "~0.11.0",
-    "peer-info": "~0.14.1"
+    "ipfs": "^0.36.4",
```

Comment:
Why does this depend on `ipfs`?

Reply:
![giphy](https://user-images.githubusercontent.com/665810/60956330-877ab380-a2fa-11e9-977d-4a52cedf983d.gif)


Oops. Have removed.